meta-microservicebus-node/recipes-microservicebus-node/nodejs-serialport/nodejs-serialport_10.4.0.bb

```diff
@@ -0,0 +1,75 @@
+# Recipe created by recipetool
```

Comment:
Remove node-serialport recipe, msb-core fetch this on demand.

Reply:
* Removed the **node-serialport** recipe in https://github.com/axians/meta-microservicebus/pull/49/commits/ec66b71e75e941b89f45fdb6c8329761356469eb
* Removed **packagegroup-microservicebus-extra** in https://github.com/axians/meta-microservicebus/pull/49/commits/7cbc4c41d07079808d5eedc865e355852d813e7c as its only dependency (node-serialport) was removed
* Edited **README** to remove occurrences of node-serialport and microservicebus-extra